test/arkane/arkaneInputTest.py

```diff
@@ -53,11 +52,10 @@
     process_model_chemistry,
 )
 from arkane.modelchem import LevelOfTheory, CompositeLevelOfTheory
+import pytest
```

Comment:
Should pytest be imported before local objects from RMG? Not sure how many files this will impact

Reply:
We could hit the whole repo with `isort` - there are jumbled imports all over the place. Agreed though that we could at least do it here with `pytest` to start.